Main/Source/game.cpp

```diff
@@ -5191,6 +5191,9 @@ festring game::GetDataDir()
 
 festring game::GetStateDir()
 {
+#ifdef USE_HOME_FOR_STATE_DIR
```

Comment:
It would have been cleaner to put this in the `#else` of the `#ifdef MAC_APP`, as it's only relevant for non-OSX Unixes, as OSX and Windows already default to `GetHomeDir()`.